cypress/e2e/companiesSpec.cy.js

```diff
@@ -0,0 +1,75 @@
+import { mockCompanies } from "../support/mockCompanies.js";
+
+describe("Companies page after logging in", () => {
+  beforeEach(() => {
+    cy.intercept("GET", "http://localhost:3001/api/v1/users/2/companies", {
+      statusCode: 200,
+      body: mockCompanies,
+      headers: {
+        "Content-Type": "application/json"
+      }
+    }).as("getCompanies");
+    
+    cy.visit("http://localhost:3000/");
+    cy.get("img[alt='Companies']").click();
+    /*
+    cy.get("#email").type("danny_de@email.com")
```

Comment:
Are we keeping this commented out code as a future template or did it slip through the cracks? Not a huge deal either way, just putting it on everyone's radar that it needs to be removed prior to production. This filtering may warrant creating a new issue on the project board.